static/sass/_patterns_media-object--snap.scss

```diff
@@ -43,17 +43,17 @@
     }
 
     .p-media-object__image {
+      height: 3rem;
       max-height: 3rem;
```

Comment:
There is probably no need for `max-` width and height if `width` and `height` are both specified with the same values.